packages/app/project.json

```diff
@@ -6,11 +6,47 @@
     "implicitDependencies": ["ui-extensions-go-cli"],
     "tags": ["scope:feature"],
     "targets": {
+      "clean": {
+        "executor": "nx:run-commands",
```

Comment:
would these be better as `run-script` executors? https://nx.dev/packages/nx/executors/run-script

(or the specific executor the command if one exists, I _suppose_ that's best?)

Reply:
That executor runs a script but not an executable distributed by an NPM package. I'd write our own executor to simplify this, but I'd do it in a follow-up PR>